bin/etest

```diff
@@ -166,10 +175,11 @@ if array_not_empty test_list ; then
 fi
 
 # Default working directory from conf file if unspecified, or ./output if not in either place
-: ${work_dir:=$(conf_get _EBASH_CONF etest.work_dir)}
-: ${work_dir:=./output}
-mkdir -p "${work_dir}"
-work_dir=$(readlink -f ${work_dir})
+: ${workdir:=$(conf_get _EBASH_CONF etest.work_dir)}
+: ${workdir:=$(conf_get _EBASH_CONF etest.workdir)}
```

Comment:
Same here. We accept older `work_dir` config option as well as newer `workdir` config